.circleci/config.yml

```diff
@@ -29,7 +33,7 @@ jobs:
       - run:
           name: List versions
           command: |
-            echo "npm: $(npm --version)"
+            echo "yarn: $(yarn --version)"
```

Comment:
I'd prefer sticking to `npm` unless yarn has clear upsides.

Reply:
... though you adding a `packing-lock.json` file to this project should help us trigger consistent builts on CI.

Reply:
@JamesCropcho BTW, `Dockerfile` and `deployment/Dockerfile` are still using `npm`.

Reply:
Hello @etpinard, I switched to Yarn because Falcon uses Yarn. (You will notice this is a recurring pattern.) So, you would only want Yarn if there are clear upsides, but you also list one clear upside to Yarn. So, would you the program to use Yarn or NPM? I am happy to do it either way; let me know!

@n-riesco 
> BTW, Dockerfile and deployment/Dockerfile are still using npm.

On the off chance it's not just a tiny oversight, `./Dockerfile` does not exist:
```
$ find . -name Dockerfile
./deployment/Dockerfile
$
```

However, I would be happy to amend the existing Dockerfile.

Reply:
> but you also list one clear upside to Yarn.

The upside being a package lock file? In that case, that's not an upside, npm has one now. Voting for `npm` here.

Reply:
I have no preference.

Reply:
@etpinard @n-riesco I have reverted to NPM.